ga4gh/datamodel/variants.py

```diff
@@ -680,20 +641,14 @@ def convertVariant(self, record, callSetIds):
                     value = value.split(',')
                 variant.info[key] = _encodeValue(value)
 
-        # NOTE: THE LABELED LINES SHOULD BE REMOVED ONCE PYSAM SUPPORTS
-        # phaseset
-
-        sampleData = record.__str__().split()[9:]  # REMOVAL
         variant.calls = []
-        sampleIterator = 0  # REMOVAL
-        if callSetIds is not None:
-            for name, call in record.samples.iteritems():
-                if self.getCallSetId(name) in callSetIds:
-                    genotypeData = sampleData[sampleIterator].split(
-                        ":")[0]  # REMOVAL
-                    variant.calls.append(self._convertGaCall(
-                        record.id, name, call, genotypeData))  # REPLACE
-                sampleIterator += 1  # REMOVAL
+        if callSetIds is not None and len(callSetIds) > 0:
```

Comment:
When could ``callSetIds`` be None here? I don't think that's a valid input. We can just ``assert`` that callSetIds is not None I think.